QEfficient/src/common.py

```diff
@@ -70,17 +70,16 @@ class QEFFCommonLoader:
     def __init__(self, *args: Any, **kwds: Any) -> None:
         raise EnvironmentError(
             f"{self.__class__.__name__} is designed to be instantiated "
-            f"using the `{self.__class__.__name__}.from_pretrained(pretrained_model_name_or_path)`")
+            f"using the `{self.__class__.__name__}.from_pretrained(pretrained_model_path)`")
     
     @classmethod
-    def from_pretrained(cls, pretrained_model_name_or_path: str, *args, **kwargs) -> QEFFBaseModel:
+    def from_pretrained(cls, pretrained_model_name: str, pretrained_model_path: str, *args, **kwargs) -> QEFFBaseModel:
```

Comment:
Actually, this already handles when you directly pass path of the downloaded model IMO, Am I missing something?